packages/ember-glimmer-template-compiler/tests/precompile-test.js

```diff
@@ -0,0 +1,16 @@
+import { precompile, compile, template } from 'ember-glimmer-template-compiler';
+
+QUnit.module(`Glimmer Precompile:`);
+
+QUnit.test('returns a string', (assert) => {
+  let str = precompile('Hello');
+  assert.equal(typeof str, 'string');
+});
+
+QUnit.test('when wrapped in a template, precompile is the same as compile', (assert) => {
+  // Simulating what happens in a broccoli plugin
+  // when it is creating an AMD module
+  let precompiled = template(precompile('Hello'));
```

Comment:
I think something around here was biting me w/ ember-performance

https://github.com/eviltrout/ember-performance/blob/master/compile-templates/index.js#L96

which is then eval'd?

https://github.com/eviltrout/ember-performance/blob/eb3f94aaefd37272a8f2a62d7b66145b81cf39bd/test-client/test-client.js#L168

Glimmer doesn't seem to expect the same treatment.